shaptools/hdb_connector/connectors/base_connector.py

```diff
@@ -34,6 +34,31 @@ class QueryError(BaseError):
     Error during query
     """
 
+class QueryResult(object):
+    """
+    Class to manage query results
+
+    Args:
+        cursor (cursor): sql cursor object
+    """
+
+    def __init__(self):
+        self._logger = logging.getLogger(__name__)
+        self.data = None
+        self.meta_data = None
+
+    def fetch_query(self, cursor):
+        """
+        Fetch query result
+
+        Args:
+            data (str): Result set returned after the query execution
+            meta_data (str): Meta data about the result set
+        """
+        self.data = cursor.fetchall()
+        self.meta_data = cursor.description
+        self._logger.info('query result: %s' % self.data)
```

Comment:
We have to fix all the log methods. 
Using `%` is not recommended:
https://www.simonmweber.com/2014/11/24/python-logging-traps.html
We should have instead:
```
self._logger.info('query result: %s', self.data)
```